start_docker_dev.sh

```diff
@@ -1,6 +1,7 @@
 HSUSER=headerservice
-TAGNAME=3.10.0-4.3.0-salobj_4.5.0
+TAGNAME=3.10.0-4.4.0-salobj_4.5.0-aths_1.1.0
 DOCKER_IMA=lsstdm/atheaderservice:$TAGNAME
+NAME=aths
```

Comment:
Is this variable necessary?  Where is it used?

Reply:
I use it for:
`dockerID=$(docker ps -qa --filter name=$NAME)`
in case I want to nuke a previous one that's running.